metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/repository/package.scala

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.interop.metricsreportgenerator
+
+import cats.implicits._
+import org.mongodb.scala.Document
+import spray.json.{JsonReader, _}
+package object repository {
+
+  case object NoDataFoundInDocument extends Throwable("No data field found in the document")
+
+  def extractData[T: JsonReader](document: Document): Either[Throwable, T] =
```

Comment:
this is included in `cqrs` module

Reply:
[same](https://github.com/pagopa/interop-be-jobs/pull/80#discussion_r1050787700)